### DISCUSSION TITLE
Why does the resource directory not recognize binary files or custom files？

### DISCUSSION THREAD

**Discussion by lelzhang** [godotengine/godot-proposals#7737] [Engine Core]
Binary files or.dll files are important for hot updating game resources, hopefully Godot will support them soon!

**Comment by timothyqiu** [comment]
These file are not resources so they are not exported by default.

But you can add them by opening the "Project > Export" dialog and add something like `*.dll` in the "Filters to export non-resource files/folders" field in Resources tab.

**Reply by lelzhang** [reply]
Yes, this is the pck export process ,How do I load binary files to memory if I want to load them? What type should binaries be in Godot?

**Reply by Zireael07** [reply]
This is not the place to ask support questions. Try the forums or Discord
